DataDog/storedog
# Set up DBM files

## Description

- Adds some commands and files for DBM
- Give the agent container a static ip (fixed web-1's connection issues)
- Updated the DBM query to hit the db harder and have more room for improvement in the lab
- Updated the README with instructions

## How to test

- Pull this branch
- in `services/frontend/site/featureFlags.config.json` find the object with `name:dbm` and set `active:true`
- run `docker-compose --profile dbm up -d`
- once all the containers are up, run `docker exec storedog-postgres-1 ./dbm_exec.sh` this will add a few things we need for dbm to the database
- run `docker restart storedog-postgres-1` to restart the postgres container
- to test the connections run this code snippet:
```bash
docker-compose exec postgres psql -h localhost -U datadog postgres -A \
  -c "select * from pg_stat_database limit 1;" \
  && echo -e "\e[0;32mPostgres connection - OK\e[0m" \
  || echo -e "\e[0;31mCannot connect to Postgres\e[0m"
docker-compose exec postgres psql -h localhost -U datadog postgres -A \
  -c "select * from pg_stat_activity limit 1;" \
  && echo -e "\e[0;32mPostgres pg_stat_activity read OK\e[0m" \
  || echo -e "\e[0;31mCannot read from pg_stat_activity\e[0m"
docker-compose exec postgres psql -h localhost -U datadog postgres -A \
  -c "select * from pg_stat_statements limit 1;" \
  && echo -e "\e[0;32mPostgres pg_stat_statements read OK\e[0m" \
  || echo -e "\e[0;31mCannot read from pg_stat_statements\e[0m"
```
- All statuses should return `OK`


You should now see your logs in DBM here https://dd-corpsite.datadoghq.com/databases/list
